codon_program10.py:5:1: error: unexpected dedent